README.rst

```diff
@@ -25,5 +25,5 @@ Disclaimer
 **********
 
 nRF Connect SDK supports development with nRF9160 Cellular IoT devices.
-nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in v1.3.1 for prototyping and evaluation.
+nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in v1.3.2 for prototyping and evaluation.
```

Comment:
Wouldn't it be a better to get some replacement of the version, so that we don't have to update so many places.
I'm assuming this will be overwritten to 1.4 once that is released?

Reply:
I tried to do that, but the problem is that the version on master is v1.3.99 - so if we use |version|, we'll end up with docs saying things like "to update to release v1.3.99, ...", which doesn't make sense since 1.3.99 is not a release.
We could probably add shortcuts like |lastrelease| = v1.3.2 and |lastrelease_tt| = ``v1.3.2`` and |lastrelease_number_tt| = ``1.3.2`` and |version_tt| = ``1.3.99``, but I'm not sure that is easier ...

Reply:
I think either that, or leave the example versions be. I have commented other places about this.

Reply:
Ok, we discussed this in the team now, and we'll use the shortcuts - however, we cannot use them in all contexts.
Regarding the examples, I generally agree, but we have gotten feedback and defects regarding this before. And this is understandable, since using old versions in examples makes the docs look outdated.
I'll update the PR with shortcuts (where possible).

Reply:
>  since using old versions in examples makes the docs look outdated

Unless it was obvious that it was only an example, like 1.15.37 :)

Reply:
Updated now.